From c14e297ba2ecc929961a2259dd53601376fd547e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Maxime=20M=C3=A9r=C3=A9?= <maxime.mere@foss.st.com>
Date: Fri, 12 Dec 2025 15:08:33 +0100
Subject: [PATCH 937/954] refactor(stm32mp1): print reset reason at a lower log
 level
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The reset reason is important information that should be printed even at
the NOTICE log level.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 9daab224d20b6822d66c971ace4c663acddcac88 refactor(stm32mp1): print reset reason at a lower log level]
Change-Id: Ic0a2ae68d21c7f63932a28fe390cf98f18dc776c
Signed-off-by: Maxime Méré <maxime.mere@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/517683
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 plat/st/stm32mp1/bl2_plat_setup.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plat/st/stm32mp1/bl2_plat_setup.c b/plat/st/stm32mp1/bl2_plat_setup.c
index 83013c9027..6978391cdb 100644
--- a/plat/st/stm32mp1/bl2_plat_setup.c
+++ b/plat/st/stm32mp1/bl2_plat_setup.c
@@ -75,7 +75,7 @@ static void print_reset_reason(void)
 		return;
 	}
 
-	INFO("Reset reason (0x%x):\n", rstsr);
+	NOTICE("Reset reason (0x%x):\n", rstsr);
 
 	if ((rstsr & RCC_MP_RSTSCLRR_PADRSTF) == 0U) {
 		if ((rstsr & RCC_MP_RSTSCLRR_STDBYRSTF) != 0U) {
-- 
2.53.0

